build/yamls/windows/containerd-with-ovs/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -2,11 +2,27 @@ $ErrorActionPreference = "Stop"
 $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
 $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
 $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+$OVSDriverDir="$mountPath\openvswitch\driver"
+$WorkDir="$mountPath\openvswitch\"
+
+# Check if OVSExt driver is already installed
+$driverStatus = netcfg -q ovsext
+if ($driverStatus -like '*not installed*') {
+  # Install OVS Driver
+  netcfg -l $mountPath/openvswitch/driver/ovsext.inf -c s -i OVSExt
```

Comment:
I am still unsure: which of our Windows container image include this file (`/openvswitch/driver/ovsext.inf`)?

Reply:
You can check here 
https://github.com/antrea-io/antrea/blob/0ff5d8ced9ee12ed6b29fc475638da31967179ac/build/images/Dockerfile.build.windows#L56
We are copying this openvswitch directory which contains this file from windows ovs image